### DISCUSSION TITLE
rebasing is not working on OHM project

### DISCUSSION THREAD

**Discussion by starman1121** [community/community#36436] [Other Feature Feedback, Questions, & Ideas] [answered]
I created new project by forking OHM.
but rebasing is not working correctly.
I just created distributor contract and set all parameters.
what should I do more?

**Comment by 0xlightfury** [comment, marked as answer]
hi, please try to set distributer contract as reward manager on treasury contract.

**Reply by starman1121** [reply]
ah~, I missed it.

**Comment by starman1121** [comment]
Thanks man, it works now
